domain-ee/ee-ep-merge-app/src/requirements.txt

```diff
@@ -1,5 +1,5 @@
 fastapi==0.103.*
-hoppy @ git+https://github.com/department-of-veterans-affairs/abd-vro.git@shared/lib-hoppy-0.2#subdirectory=shared/lib-hoppy
+hoppy @ git+https://github.com/department-of-veterans-affairs/abd-vro.git@shared/lib-hoppy-0.3#subdirectory=shared/lib-hoppy
```

Comment:
Dont forget to tag the commit with `lib-hoppy-0.3`, and the version stored in the file `shared/lib-hoppy/hoppy/__init__.py` should be updated as well.

Reply:
Yup, will do!